macros/APIListAlpha.ejs

```diff
@@ -48,12 +68,30 @@ function buildTitle(base) {
     // before the first space
 
     if ((spaceIndex < base.length-1) && (base[spaceIndex+1] == '(')) {
-        return "<code>" + base.substring(0, spaceIndex) + "</code>" + base.substring(spaceIndex);
+        return [
+            `<code>${
+                mdn.escapeHTML(base.substring(0, spaceIndex))
+            }</code>${
+                mdn.escapeHTML(base.substring(spaceIndex))
+            }`,
+            base,
+        ];
+    }
+
+    const match = /^(?:API )?([^ ]+)(?: (?:-|Interface))?/i.exec(mdn.escapeHTML(base));
```

Comment:
This ensures that modern page names won’t break:
- https://discourse.mozilla.org/t/incorrect-titles-for-method-property-articles/31641
- https://discourse.mozilla.org/t/api-documentation-structure-idea/31967
- https://discourse.mozilla.org/t/brainstorming-updating-titles-of-css-reference-pages/23126
- https://discourse.mozilla.org/t/proposal-changing-the-titles-of-html-element-reference-pages/19112

Reply:
IMO this should instead be using the slug to generate the name of the CSS item, rather than trying to strip it from the title.

Reply:
This is about APIs, not CSS.

Reply:
The fact remains that trying to strip the name of the thing out of the title is not going to be as reliable as getting it from the slug.

Reply:
Which this is. Dammit, never mind.